### PR TITLE
feat: [TKC-1948] disable webhooks

### PR DESCRIPTION
**Pull request by vLia**
## Pull request description 

This PR adds the option to disable webhooks and makes sure that no notification is sent out on a disabled webhook.

## Checklist (choose whats happened)

- [ ] breaking change! (describe)
- [x] tested locally
- [ ] tested on cluster
- [ ] added new dependencies
- [x] updated the docs
- [x] added a test

## Breaking changes

-

## Changes

-

## Fixes

-

### CODE REVIEW

File: pkg/event/kind/webhook/listener.go [hunk 1]
```diff
@@ -83,7 +86,16 @@ func (l *WebhookListener) Headers() map[string]string {
...
 }
 
+func (l *WebhookListener) Disabled() bool {
+	return l.disabled
+}
+
 func (l *WebhookListener) Notify(event testkube.Event) (result testkube.EventResult) {
+	if l.disabled {
+		l.Log.With(event.Log()...).Debug("webhook listener is disabled")
+		return testkube.NewFailedEventResult(event.Id, errors.New("webhook listener is disabled"))
```
Comment (vsukhin) [inline pkg/event/kind/webhook/listener.go:96]: do we need to return an error?

**Review by vsukhin — APPROVED** [review]
one question

File: pkg/event/kind/webhook/listener.go (hunk 1 above)
Comment (vLia) [reply pkg/event/kind/webhook/listener.go:96]: The notification failed, even if it was expected to fail, that is why I chose to return a failed event result

Comment (vsukhin) [reply pkg/event/kind/webhook/listener.go:96]: I would not report an error, because we expect, it would not work

**Review by jfermi — APPROVED** [review]

File: pkg/event/kind/webhook/listener.go (hunk 1 above)
Comment (vLia) [reply pkg/event/kind/webhook/listener.go]: Makes sense. I updated it.

**Pull request merged**